keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/MetropolisHastings.java

```diff
@@ -77,6 +78,13 @@ public NetworkSamples getPosteriorSamples(BayesianNetwork bayesianNetwork,
             .generate(sampleCount);
     }
 
+    public NetworkSamples getPosteriorSamples(BayesianNetwork bayesianNetwork,
+                                              Vertex vertexToSampleFrom,
+                                              int sampleCount) {
+        return generatePosteriorSamples(bayesianNetwork, Arrays.asList(vertexToSampleFrom))
```

Comment:
is it worth considering Collections.singletonList() instead of asList()?

Reply:
👍

Reply:
cool changed to `singletonList`